modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -33,6 +33,17 @@
 public enum AndesConfiguration implements ConfigurationProperty {
 
     /**
+     * The coordination type. If coordination type is set to "HAZELCAST" cluster communication will be handled
+     * through Hazelcast messages where as if coordination type is set to "RDBMS", it'll be via the database
+     */
+    CLUSTER_EVENT_SYNC_MODE("coordination/clusterCommunicationType", "RDBMS", String.class),
```

Comment:
suggestion make the constant align with the actually configuration names.
e.g. CLUSTER_EVENT_SYNC_MODE("coordination/eventSyncMode" ( - less preffered to me)
CLUSTER_COMMUNICATION_TYPE("coordination/clusterCommunicationType)